### ISSUE TITLE
toDataURL quality range

### ISSUE THREAD

**Issue by rambo-panda** [Brooooooklyn/canvas#498]
> encoderOptions Optional
A [Number](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number) **between 0 and 1 **indicating the image quality to be used when creating images using file formats that support lossy compression (such as image/jpeg or image/webp). A user agent will use its default quality value if this option is not specified, or if the number is outside the allowed range.
>
> https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/toDataURL

https://github.com/Brooooooklyn/canvas/blob/v0.1.25/src/lib.rs#L362-L367
```rust
  let quality = match ctx.get::<JsNumber>(1)?.get_uint32() {
    Ok(number) => number as u8,
    Err(_e) => match mime {
      MIME_WEBP => DEFAULT_WEBP_QUALITY,
      _ => DEFAULT_JPEG_QUALITY, // https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/toDataURL
    },
```

Should browser float rules be compatible?
in other words
Should it be consistent with web(browser) rules?

**Comment by Brooooooklyn** [comment]
yes, it should align to the browser API
